src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/_advanced_parameters.yml

```
@@ -45,3 +45,7 @@ _employee:
 _profiles:
     resource: "profiles.yml"
     prefix: /profiles
+
+_experimental_feature:
+    resource: "experimental_feature.yml"
+    prefix: /experimental-feature
```

If you choose to keep "experimental features" you will need to add the final "s"
```suggestion
_experimental_features:
    resource: "experimental_features.yml"
    prefix: /experimental-features
```

---

Every file in this folder is singular, not plural https://github.com/PrestaShop/PrestaShop/tree/develop/src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters

However you are right about the prefix, it must be plural to fit with other routes

---

The line above reads "profiles", so Your Argument Is Wrong 😛

---

Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/a95d4af99ad89a142b320d0011ddb9db2e5270cf